docs/reference/reference-deploy/config/master-config-reference.rst

```diff
@@ -113,6 +113,10 @@ The master supports the following configuration settings:
       ``/var/cache/determined``. Note that the master would break on startup if it does not have
       access to create this default directory.
 
+-  ``launch_error`` (optional): Specify whether to refuse an experiment or task if the slots
+   requested exceeds the cluster capacity. If ``false``, only a warning will be returned. Defaults
+   to ``true``.
+
```

Comment:
-  ``launch_error`` (optional): Specifies whether to refuse an experiment or task if the slots
   requested exceeds the cluster capacity. If set to ``false``, only a warning is returned. The default value is ``true``.

also, is it "the cluster capacity" or the "cluster's capacity"?

Reply:
I think it should be the "cluster's capacity"